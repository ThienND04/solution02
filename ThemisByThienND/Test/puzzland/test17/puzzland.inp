17
14 54
Q Y V D C H M P E R S F X T 
V E
M R
H F
F T
V F
R S
Y R
Y P
Q F
Q D
H S
M F
C T
C F
C X
D S
H P
C M
D X
P R
D C
Y T
P S
V X
Q S
P X
C P
C H
Y S
Y H
Q T
D F
V C
V T
P E
Q E
C E
V S
S X
V D
R T
Q M
E T
Q V
S T
M S
Q R
Y C
M E
M P
E S
Q C
S F
Q X
12 54
L N H I M U C F Z T K P 
I F
N F
C T
N C
I U
C P
C K
M K
N U
L P
N I
L F
C F
M T
L N
L U
U P
C Z
Z T
U K
T P
N M
I Z
Z P
M C
N P
U Z
M Z
M F
L C
N Z
Z K
L Z
U C
H P
F T
N T
K P
H K
N K
U T
H M
L K
H T
I T
T K
H C
M U
U F
H F
I P
N H
H Z
I M
14 87
D Q I X C G F N W Y S H O L 
F H
H O
S O
N W
N H
D Y
D N
X L
I O
X N
N L
D C
I W
Q L
I S
C Y
D S
C G
I N
G Y
G S
O L
X G
C F
D G
D I
I C
F Y
Q W
S L
D O
G F
X S
X C
C H
G W
W S
Q F
C N
N Y
I Y
X O
Q H
Q C
D X
C O
C S
G O
I X
Q S
X W
Q X
W Y
X Y
D W
I L
D Q
Q I
N O
W H
F L
H L
X H
Y O
D L
Q O
C L
F N
G N
Q N
S H
Y H
X F
C W
F S
F W
F O
D F
I H
G L
I F
Y L
I G
W O
Q Y
N S
W L
14 75
S M O V E N U W T B Z Y Q G 
M N
T Y
E T
W G
V N
Z Y
N Z
B Q
O E
S W
Z Q
M T
V W
M U
M Q
V T
E W
O U
W Z
E Y
N W
B Z
M B
O V
Y G
E N
V Q
M V
T G
V G
O G
V B
E U
E B
S Q
U W
W B
T Z
M Z
W Q
S E
S Y
S B
E G
S M
M G
M W
S T
T Q
S G
S V
E Z
O Z
M O
W Y
U Q
U T
U Z
V U
S N
T B
O T
N U
M E
S Z
Q G
V Z
E Q
V Y
O Y
Y Q
O B
U G
V E
O W
12 51
D F O L C P B K W Z G T 
L W
O C
F L
F B
P W
D O
L C
B T
W Z
O G
F C
G T
D L
D W
B Z
L P
F K
B G
B K
Z T
Z G
P B
O K
K T
O Z
O T
F G
F W
C W
L G
C P
O W
P T
O P
P K
D P
D G
F T
P Z
F P
D K
L K
P G
O B
F O
F Z
K W
K G
O L
D F
D T
12 53
T E Z I D J K O W A H N 
J W
D W
D N
Z D
E I
J H
K O
K W
O H
I H
T D
I A
W A
O N
E H
I O
D O
K N
D H
Z K
T H
D K
I J
T W
Z A
E Z
E D
T I
O W
E J
A H
I K
Z N
T A
Z W
D J
E O
T J
D A
E W
T E
W H
I D
W N
T N
K A
T K
A N
Z J
K H
Z I
E K
I W
15 78
V D H Q E C Z J I N U X G F A 
Z U
Q E
E G
I N
E I
C Z
N U
D G
V N
H J
D I
I A
J G
Z N
J U
C N
D H
H N
U A
E A
E C
H F
C J
N X
H X
D Z
Q Z
Z G
J A
E U
Q N
H U
V E
V F
H A
N F
V Z
Z I
D A
E J
Q I
E Z
C U
Z A
D N
H Q
G F
N G
H Z
E X
G A
N A
I U
V I
D F
D J
D Q
H G
C A
V A
E F
H I
C G
Q X
D E
H C
C F
Q U
U X
V Q
Q F
Z J
C X
Q J
E N
Q C
V X
X A
14 88
Z C J K H L R Q X T G Y M F 
K L
L F
K H
C G
Y F
L G
C T
L T
M F
J L
Z H
H M
K G
C K
K R
G F
J Y
L R
R F
H Q
J F
H T
L X
Y M
J H
Z C
C F
J R
R M
Q G
R X
K M
H R
R G
Z F
H X
C R
R Y
H L
K Y
T M
Z J
Z M
Z Q
H Y
X F
Z L
R T
X G
T F
Q Y
K Q
Q T
X M
C X
L Q
J K
R Q
T Y
K X
Z Y
Q M
H F
J X
X Y
C L
L M
C H
Z G
Q X
C Q
T G
Q F
C J
L Y
J M
G Y
C Y
J Q
Z T
X T
J G
K T
K F
Z R
C M
Z X
J T
14 62
B R K I M A C J X P S G W Y 
P G
K X
X Y
R W
R P
K A
I C
K C
R G
B G
K J
B R
G Y
K Y
R S
M Y
I S
K W
A P
P Y
J W
C S
J S
I G
A J
B I
P S
M C
I J
B P
C X
A G
M P
B A
B C
K G
S Y
I W
S W
S G
R A
M G
I Y
M X
R M
X G
A W
C Y
C G
R K
K M
I X
W Y
J X
X P
B M
J P
R Y
M W
B K
C J
C W
14 85
B F W O L R G P I Q A D C H 
G I
I H
R Q
B W
B I
O C
W O
Q D
L A
G H
R H
B O
D H
B P
F H
L D
F R
C H
P C
W D
B C
O I
W G
B A
R G
W H
R C
F A
O A
G A
B G
W Q
A D
L P
O R
F G
W I
L Q
R I
G C
B H
O D
W R
P Q
O G
F P
F Q
P I
Q H
L C
L I
G P
L G
W P
W L
I D
F O
A H
W C
B Q
A C
B F
R A
R D
L R
L H
D C
R P
O Q
F C
P D
O L
F L
G D
P H
O H
B R
F I
G Q
W A
P A
F W
I Q
I A
B L
15 67
S A V K Y B D F G Q T R C Z M 
T C
V R
R C
B F
K Y
K T
Z M
A R
Q M
Y T
A V
Y C
K R
B G
S V
D M
K G
S Q
Y D
Q R
V M
Y G
A G
C M
D T
G M
K Q
A D
B D
B T
D G
A F
G T
D F
Y Z
S D
S Y
S B
V F
S T
V Q
A Y
B M
Y M
S A
V Y
F C
V B
S F
K C
F Z
K Z
T Z
G C
V T
Y F
C Z
Y R
V D
A Q
F T
Y B
K B
V G
B Q
Q C
S K
14 75
J B L G Y T V D X F S E R Q 
T Q
F R
J S
J L
F S
J G
J T
Y S
J Q
E R
T F
L Q
G V
S Q
V S
X R
F E
B Q
G Q
F Q
L V
V X
Y V
D S
V Q
G F
T R
Y X
L Y
B X
X Q
L E
B L
X E
L D
B F
B G
T D
J Y
G R
L T
G X
V F
J V
T E
V D
E Q
L F
Y D
R Q
X S
D X
G S
B S
Y E
V R
S R
J E
L X
G D
B E
S E
J R
D R
T S
B V
G Y
J B
Y T
Y R
D E
B R
Y F
Y Q
G E
13 53
W O D F Y P Z A B I Q J N 
A J
F Q
Y A
A Q
Y B
A B
D A
O Z
Y P
D N
W D
Q N
I N
D B
O P
O F
W A
Z Q
D I
F N
W Y
Z J
P I
O B
D P
W Z
D Z
Y I
A I
P A
B J
I J
Q J
O A
W P
Z B
F J
F P
P J
Y Z
A N
Y J
O D
W I
D F
W N
P Z
W O
P N
O Q
W Q
W B
D J
12 49
M Y D V P C E X O N S R 
X S
Y R
M Y
V C
D R
Y S
C X
V S
O S
D S
M V
D C
P C
Y V
X R
P E
P X
V O
M N
M R
E S
Y C
Y O
M D
X O
M X
C O
P S
D X
M C
V P
P R
N S
D O
D V
E X
O N
M E
X N
V N
E N
Y P
P O
M P
O R
V X
Y N
D N
Y X
14 62
I O U S X T D L C H M A P B 
O A
H B
I H
X A
D M
O P
P B
H M
U P
D B
T P
O H
U B
T H
I P
T D
I C
I X
T L
X B
S T
I L
L H
O X
U X
T A
X M
I T
T C
O S
M A
C B
S A
O D
X T
O C
H A
A P
O T
L C
U L
D H
L P
S P
S B
I D
D C
X L
X P
X D
U A
I B
I A
S D
M B
S L
U S
I M
X C
C P
O M
S H
12 56
L X M U F V Y Z N O S A 
Z O
X U
Z A
F Z
M Y
M Z
L S
X A
X O
Z N
V O
Y Z
M U
Z S
V Y
X N
M O
U N
Y N
M A
F V
L Z
L A
U Z
O S
M F
M N
V Z
U A
L Y
F Y
U V
N O
N S
V N
X M
L X
N A
U S
L N
F N
X Z
L U
Y O
Y S
M S
O A
M V
V S
X Y
Y A
X S
L F
V A
U O
S A
12 66
I E V Q Z Y X B U W R H 
Z R
E R
X W
Z H
I E
I V
E Q
V Z
Z B
Q R
V U
E W
B R
E B
W R
Q X
Y W
V W
X R
U R
R H
U H
I W
X H
Z Y
Y R
Q W
E Y
Q B
E U
I H
W H
Y X
X B
I X
Q Y
X U
E V
V H
V Q
Y U
E X
I Y
B H
Q Z
V X
Y H
B W
V R
Y B
Z U
Q H
Q U
E Z
V B
B U
Z X
I R
Z W
V Y
U W
I B
I Z
I Q
I U
E H
