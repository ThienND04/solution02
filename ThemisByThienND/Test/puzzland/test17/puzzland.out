QCDFHPEMRYSXVT
LCFHKMINTUZP
DCFGIHNOQSWXYL
SBEMNUOTQVWZYG
DFBGKLCOPWZT
TADEHIJWOKZN
VECFDGHINQJZUXA
ZCGJHKLMQRTXYF
BAGCIJKMRWSPXY
BACDGFILOQPRWH
SADBFCGKQRVTYZM
JBEDGFLTRSVXYQ
WABDFJIPYZOQN
MCDNEPOSVXYR
IAHDCLPOMXTSUB
LFMNOSUVYXZA
IBEQRUVWXYZH
